docs/installation.md

```diff
@@ -326,6 +326,13 @@ python3 -m pip install --upgrade pip
 python3 -m pip install -e .
 ```
 
+Patch conda libta-lib
+
+```bash
+cd build_helpers
+bash install_ta-lib_conda.sh
```

Comment:
I don't really see this as applicable to a conda environment.

To me, there is no reason to use conda in a linux/macos environment (where python / build dependencies are easy to install) - so the main usecase for conda is windows - where installation is simplified to some degree.
As such, a linux only script provides no benefit.

One benefit would be to have a windows script for this - but you don't really want to go there (build dependencies on windows are a pain), and instead reuse the available windows wheels.

For linux however, installing ta-lib can also be done via the available script - and will work just fine for a conda environment.

Reply:
OK, should the documentation be updated to tell the conda user how to patch ta-lib? To prevent new user confusion.

Reply:
btw you can close this PR.